.gitignore

```diff
@@ -66,3 +66,6 @@ src/config.json
 
 # generated fcc curriculum data file
 src/commands/scrapers/data/fcc/challengeData.json
+
+# editor specific settings
+.vscode
```

Comment:
Also, you can leave this in. I don't use vscode but for those that do it will at the least launch the bot via the editor run. VSCode intends for the `.vscode/launch.json` to be committed to repos as it only contains shareable contents. That way you can share debugging and launching settings.

Reply:
Also, I'd like to add, this isn't relevant to the description or issue you set out to fix. Keeping pull requests contained is best for getting them accepted on most projects. Instead of trying to make the executive decision to remove this from tracking, create an issue for it to be discussed.

Also, maybe do an interactive commit and only add the hunks you need instead of touching the entire file in pull requests. That way merge conflicts can be more easily avoided.

Otherwise, your editor seems to be touching the entire file with respects to quotes and indents. Since we've not established a style guide and there were previous commits allowed through that didn't stick with one style over others in respect to things like quotes, I think your commit could set things right. So in this instance it seems fine. However in the future, consider adhering to the style initially in the file. If you're fighting your editor the whole way then again interactive commits are good solutions.